packages/chat-component/src/main.ts

```diff
@@ -647,22 +706,35 @@ export class ChatComponent extends LitElement {
                   </ul>
                 </nav>
                 <div class="aside__content">
-                  <div id="tabpanel-1" class="aside__tab active" role="tabpanel" tabindex="0" aria-labelledby="tab-1">
+                  <div id="tabpanel-1" class="aside__tab${
+                    this.selectedAsideTab === 'tab-thought-process' ? ' active' : ''
+                  }" role="tabpanel" tabindex="${
+                    this.selectedAsideTab === 'tab-thought-process' ? 0 : -1
+                  }" aria-labelledby="tab-thought-process">
                     <h3 class="subheadline--small">${globalConfig.THOUGHT_PROCESS_LABEL}</h3>
                     <div class="aside__innerContainer">
                     ${this.chatThoughts ? html` <p class="aside__paragraph">${unsafeHTML(this.chatThoughts)}</p> ` : ''}
                     </div> 
                   </div>
-                  <div id="tabpanel-2" class="aside__tab" role="tabpanel" aria-labelledby="tab-2" tabindex="-1">
+                  <div id="tabpanel-2" class="aside__tab${
+                    this.selectedAsideTab === 'tab-support-context' ? ' active' : ''
+                  }" role="tabpanel" aria-labelledby="tab-support-context" tabindex="${
```

Comment:
using aria-hidden or hidden property in the tab panels would be for accessibility better than using tabindex as div are not focusable by default

Reply:
Tabindex and aria-hidden are complementary not exclusive. Aria hidden is for the end-user to understand there is content not visible and tabindex is to tell assistive tech that content can be focusable. This divs are meant to be focusable and hence you change the tab-index from -1 to 0 and vice-versa

Reply:
What happens is that when you activate the tab, you also activate the associated content, enabling the focus on the panel.

Reply:
In any case, we should have a specific story to review accessibility at a component wide level. I have a couple of friends who specialize on the subject and can ask them for a review.